<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>484</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Orange Music Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/music.png</normaloff>Icons/music.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton_play">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/play.png</normaloff>
      <normalon>Icons/pause.png</normalon>
      <disabledoff>Icons/play.png</disabledoff>
      <disabledon>Icons/pause.png</disabledon>Icons/play.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_previous">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/previous.png</normaloff>Icons/previous.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_next">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/next.png</normaloff>Icons/next.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_playlist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/local.png</normaloff>Icons/local.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_play_single">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/single.png</normaloff>Icons/single.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_position">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_playlist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>播放列表</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_playlist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_pause">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/pause.png</normaloff>Icons/pause.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_stop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/stop.png</normaloff>Icons/stop.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_start_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>4</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_stop_time">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>390</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>4</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_time">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>4</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_play_order">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/order.png</normaloff>Icons/order.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_play_random">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/radom.png</normaloff>Icons/radom.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_speed">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>× 2</string>
    </property>
    <item>
     <property name="text">
      <string>× 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>× 1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>× 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>× 0.5</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>405</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
